app/views/conservation_records/_form.html.erb

```diff
@@ -14,17 +14,17 @@
 
   <div class="form-block">
     <div class="form-group">
-      <%= form.label :date_received_in_preservation_services %>
+      <%= form.label :date_received_in_preservation_services %> <span style="color: #e00122;">*</span>
       <%= form.date_field :date_received_in_preservation_services, class: 'form-control' %>
     </div>
 
     <div class="form-group">
-      <%= form.label :department %>
+      <%= form.label :department %> <span style="color: #e00122;">*</span>
```

Comment:
To DRY this up a bit, can you move the color specification to application.scss and just use something like `<span class="required">*</span>` in the views?  That puts the color code in a single place and makes it easier to change down the road.

There are also ways to use CSS pseudo-classes to automatically add an asterisk to required field labels, but that's probably getting more complicated that it needs to be.